std/string.d

```diff
@@ -1193,6 +1236,37 @@ ptrdiff_t lastIndexOf(Char1, Char2)(const(Char1)[] s, const(Char2)[] sub,
     return -1;
 }
 
+/// Ditto
+ptrdiff_t lastIndexOf(Char1, Char2)(const(Char1)[] s, const(Char2)[] sub,
+        in size_t startIdx, in CaseSensitive cs = CaseSensitive.yes) @safe pure
```

Comment:
0e1afe82 introduced a bug. all (last,any)indexOf should return -1 if the needle can't be found and you can't find nothing.